cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java

```diff
@@ -119,6 +119,9 @@
       .put(Constants.Metrics.Tag.FLOWLET, "flowlet")
       .put(Constants.Metrics.Tag.FLOWLET_QUEUE, "queue")
 
+      .put(Constants.Metrics.Tag.QUEUE_PRODUCER, "producer")
```

Comment:
Does it make sense to have generic "PRODUCER" and "CONSUMER" tag? to be reused, e.g. like HANDLER?